src/gdal.mk

```diff
@@ -87,4 +87,10 @@ define $(PKG)_BUILD
     $(MAKE) -C '$(1)/ogr'   -j '$(JOBS)' install OGR_ENABLED=
     $(MAKE) -C '$(1)/apps'  -j '$(JOBS)' install
     ln -sf '$(PREFIX)/$(TARGET)/bin/gdal-config' '$(PREFIX)/bin/$(TARGET)-gdal-config'
+
+    mkdir -p '$(PREFIX)/$(TARGET)/bin
```

Comment:
Probabably `mkdir` is not needed here. I just want to be safe, since [many packages](https://github.com/mxe/mxe/issues/1111#issuecomment-169280181) are broken because of trying to copy `test-foo.exe` to missing `usr/<target>/bin/`.